modules/ROOT/pages/time-zone.adoc

```diff
@@ -0,0 +1,57 @@
+= Configuring time zone
+
+By default, Fedora CoreOS machines keep time in the Coordinated Universal Time (UTC) zone and synchronize their clocks with the Network Time Protocol (NTP). This page contains information about customizing the time zone.
+
+== Viewing and changing time zone
+
+The `timedatectl` command displays and sets the date, time, and time zone.
+
+[source,bash]
+----
+$ timedatectl status
+               Local time: Mon 2021-05-17 20:10:20 UTC
+           Universal time: Mon 2021-05-17 20:10:20 UTC
+                 RTC time: Mon 2021-05-17 20:10:20    
+                Time zone: UTC (UTC, +0000)           
+System clock synchronized: yes                        
+              NTP service: active                     
+          RTC in local TZ: no
+----
+
+You can use the `list-timezones` subcommand to list the available time zones. Available time zones are represented by https://man7.org/linux/man-pages/man5/tzfile.5.html[`tzfile`] entries in the system's time zone database under `/usr/share/zoneinfo`.
+
+[source,bash]
+----
+$ timedatectl list-timezones
+Africa/Abidjan
+Africa/Accra
+Africa/Addis_Ababa
+…
+----
+
+See the https://www.freedesktop.org/software/systemd/man/timedatectl.html[manual page] for more information about how `timedatectl` can be used; however, we do not recommend changing the time zone per-machine imperatively via SSH.
+
+=== Recommended time zone: Coordinated Universal Time (UTC)
+
+To avoid time zone confusion and the complexities of adjusting clocks for daylight saving time (or not) in accordance with regional custom, we recommend that all machines in Fedora CoreOS clusters use UTC. Setting all machines to a single time zone will also help with correlating log files across systems. UTC is the default time zone.
+
+=== Setting the time zone via Ignition
+
+If your applications require a different system time zone, you can do so by setting the local time zone configuration file, https://www.freedesktop.org/software/systemd/man/localtime.html[`/etc/localtime`], to be an absolute or relative symlink to a `tzfile` entry under `/usr/share/zoneinfo/`.
```

Comment:
In most cases, you can force a timezone for just some application via environment variables: https://wiki.archlinux.org/title/System_time#Per-user/session_or_temporary_settings